src/toil/common.py

```diff
@@ -132,6 +133,24 @@ def f(x):
         setOption("masterIP") 
         setOption("parasolCommand")
         setOption("maxParasolBatches", int, iC(1))
+
+        def splitEnvVarsToDict(envVars):
+            """
+            Parses of list of environment variables and values. Names and their values are to be
+            separated by a space, an entirely different variable is to be separated by a comma.
+            e.g. --setEnv var1 val1, var2, var3 val3
```

Comment:
If we were to go with this method (which I don't think we need to), I would want to see comprehensive doctests on it.